src/main/java/core/basesyntax/service/ValueOfFruit.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public interface ValueOfFruit<T> {
+    T valueOf(String fruit, int amount);
+
+    default List<T> valueOf(Map<String, Integer> fruits) {
+        return fruits.entrySet().stream()
+                .map(e -> valueOf(e.getKey(), e.getValue()))
+                .collect(Collectors.toList());
+    }
```

Comment:
do we need to parametrize it or do we really need this class at all? it looks strange from any point of view